src/lwc/rd2PauseForm/rd2PauseForm.js

```diff
@@ -44,8 +46,11 @@ export default class Rd2PauseForm extends LightningElement {
     recordName;
 
     @track isLoading = true;
-    @track hasAccess = true;
-    @track isRDClosed;
+    @track permissions = {
+        hasAccess : false,
+        isBlock : false,
```

Comment:
- Let's rename `isBlock` to `isBlocked`.
- The `blockReason` can stay as is or probably it should be renamed to `blockedReason`?